readme.md

```diff
@@ -432,6 +432,20 @@ is.any(is.boolean, 'unicorns', [], new Map());
 //=> false
 ```
 
+See below for use with multiple predicates.
```

Comment:
If it's preferred, I can document both usages in the same block, perhaps like `.any(predicate|predicates[], ...values)`

Reply:
I see a thumbs-up, should I put them in the same block?

Reply:
Yup. Just keep readable spacing, as in `.any(predicate | predicates[], ...values)`

Reply:
Actually, I just saw that you separated single and multiple predicates to different blocks. I think it's better. Keep it as is

Reply:
Oh sorry, I updated it before seeing this latest comment. Please let me know what I should do.